src/conversion/modelconversiondispatcher.js

```diff
@@ -534,10 +510,15 @@ export default class ModelConversionDispatcher {
 	 * * there is only one event,
 	 * * consumables object includes marker range with event name.
 	 *
+	 * If selection inside a marker is converted:
+	 * * the is only one event,
```

Comment:
there

Reply:
And I don't get what does this sentence mean here.

Reply:
It's in the same vein as above. It means that there is just one `attribute:` event, not multiple, like when non-collapsed range is converted.

Reply:
Please, please, please, pretty please – a paragraph should be followed by a blank line. Write markdown just like you'd like it to render (and you always have a margin between a paragraph and a list).

Reply:
They feel so disconnected though :O.

Reply:
That's how it should be. If they miss each other they can send each other letters.